app/client/src/components/InstallChaos/styles.ts

```diff
@@ -34,4 +34,142 @@ export const useStyles = makeStyles((theme: Theme) => ({
 	done: {
 		color: theme.palette.success.dark,
 	},
+	statusheading: {
```

Comment:
Some inconsistent in name, It should be `statusHeading`

Reply:
Check for the other also

Reply:
yeah i am on it..